WalletWasabi.Gui/Shell/Commands/WalletCommands.cs

```diff
@@ -16,7 +16,7 @@ internal class WalletCommands
 		public WalletCommands(CommandIconService commandIconService)
 		{
 			GenerateWalletCommand = new CommandDefinition(
-				"Generate Wallet",
+				"Generate New Wallet",
```

Comment:
NACK, it makes it longer, the spelling is correct and the function is not ambiguous.